yarn.lock

```diff
@@ -15115,7 +15105,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"debug@npm:^3.0.0, debug@npm:^3.1.0, debug@npm:^3.1.1, debug@npm:^3.2.7":
+"debug@npm:^3.0.0, debug@npm:^3.1.0, debug@npm:^3.1.1, debug@npm:^3.2.6, debug@npm:^3.2.7":
```

Comment:
This is an example of a "deduped" dependency in yarn.lock. If anything requests any of these five versions of `debug`, yarn will resolve it to version 3.2.7, thus avoiding extra copies of the dependency on the file system